bin/infra

```diff
@@ -82,6 +88,7 @@ Command Runner for syncm8 infra commands.
 Commands:
   build       Build dev image.
   login       Authenticate docker with aws ecr.
+  priv        Run command with aws authentication enabled.
```

Comment:
qq, what are some example use cases, and is there a better name privileged? I feel like it could be a bit of a misnomer, but Idk where exactly it'd be used.

Reply:
one off debugging for developing new aws shell/ github action commands. I used it when testing the `aws ecs` command in dev/prod and the gh action.
A different name might be aws-priv, but since I expect it to almost never get used again, am tempted to keep it the same.